### ISSUE TITLE
Culture sensitive issue

### ISSUE THREAD

**Issue by latop2604** [kevingosse/GummyCat#12]
Hello, using a french culture, the app does not work. It stay blank after attaching a process.

The issue is in Region.axaml.cs. `TransformOperations.Parse($"scaleX({Width * GetFillFactor(segment)})")` fail because the interpolated string is generated as `scaleX(6,555)` instead of `scaleX(6.555)`.

I was able to fix the issue using `FormattableString.Invariant` like this

`TransformOperations.Parse(FormattableString.Invariant($"scaleX({Width * GetFillFactor(segment)})")));`

It also work using `<UseSystemResourceKeys>true</UseSystemResourceKeys>` in csproj

**Comment by kevingosse** [comment]
Good catch. I added `<InvariantGlobalization>true</InvariantGlobalization>` in the csproj, I believe this should fix your issue.
